lib/BigML.js

```diff
@@ -75,6 +75,11 @@ function BigML(username, apiKey, context) {
   this.project = undefined;
   this.organization = undefined;
   this.resourceUrls = {};
+  this.debug = false;
+  try {
+    this.debug = (process.env.BIGML_DEBUG == "1");
+  } catch (e)
+  {}
```

Comment:
just asking: is this an idiomatic way of checking whether env has the key?
(in other languages one just checks if it's present, without exception handling)

Reply:
The try/catch is meant to keep the code crossplatform. Other platforms will not offer a process object that allows us to access the environment variables. We need to avoid failing there.